evergreen/index.html

```diff
@@ -192,10 +192,10 @@ <h2>Plant Chat</h2>
                     <form id="uploadForm" method="post" enctype="multipart/form-data">
                         {% csrf_token %}
                         Image:
-                        <input id="postImage" type="file" name="image_upload">
+                        <input id="postImage" type="file" name="image_upload" onsubmit="event.preventDefault()";>
                         <br>
                         Caption:
-                        <input id="postCaption" type="text" name="caption">
+                        <input id="postCaption" type="text" name="caption" onkeypress="handleEnterKey(event)">
                         <button type="button" onclick="uploadPost()">Upload</button>
```

Comment:
Change the uploadPost to `type="submit"` and replace `method="post"` with `action="uploadPost()"` on the `form`

https://www.w3schools.com/tags/att_input_type_submit.asp

Reply:
This breaks upload functionality (both the part and the whole)